develop/parachain-devs/interoperability/transfer-assets-with-xcm.md

```diff
@@ -0,0 +1,181 @@
+- You have opened the message passing channel to allow communication between parachain B and parachain A
+- You have set up a parachain test network using Zombienet or a local relay chain using the `rococo-local` chain specification. This tutorial uses the following Zombienet configuration file:
+
+    ```toml
+    [relaychain]
+    default_command = "polkadot"
+    chain = "rococo-local"
+
+    [[relaychain.nodes]]
+    name = "alice"
+    validator = true
+    ws_port = 9944
+
+    [[relaychain.nodes]]
+    name = "bob"
+    validator = true
+    ws_port = 9955
+
+    [[parachains]]
+    id = 1000
+
+    [parachains.collator]
+    name = "parachainA"
+    ws_port = 9988
+    command = "parachain-template-node"
+    args = [ "-l=xcm=trace" ]
+
+    [[parachains]]
+    id = 1001
+
+    [parachains.collator]
+    name = "parachainB"
+    ws_port = 9989
+    command = "parachain-template-node"
+    args = [ "-l=xcm=trace" ]
+
+    [settings]
+    timeout = 1000
+    ```
+
+    !!! note
+        Due to a well-known issue with Zombienet and HRMP channels, the opening of the HRMP channels between parachains has been forced by the relay chain through the `sudo` pallet for this tutorial. For further information about this workaround, please refer to [https://github.com/paritytech/polkadot-sdk/pull/1616#issuecomment-1727194584](https://github.com/paritytech/polkadot-sdk/pull/1616#issuecomment-1727194584){target=\_blank}
+
+## Configure XCM Instructions
+
+To illustrate the interaction between the two chains, in the following example, parachain B sends XCM instructions to deposit assets into an account on parachain A.
+
+1. Navigate to the **Extrinsics** tab in the Polkadot.Js Apps interface
+      1. Select the **Developer** tab
+      2. Choose the **Extrinsics** option from the dropdown menu
+
+        ![](/images/tutorials/polkadot-sdk/build-a-parachain/transfer-assets-with-xcm/transfer-assets-with-xcm-1.webp)
+
+2. Use the **sudo** pallet to send the XCM instructions
+      1. Select the **sudo** pallet from the dropdown menu
+      2. Choose the **sudo(call)** extrinsic to use the Sudo pallet to execute privileged transactions
+
+        ![](/images/tutorials/polkadot-sdk/build-a-parachain/transfer-assets-with-xcm/transfer-assets-with-xcm-2.webp)
+
+3. Select the **polkadotXcm** pallet to send the XCM instructions
+      1. Select **polkadotXcm** pallet from the dropdown menu
+      2. Select **send(dest, message)** extrinsic to send a Cross-Chain Message (XCM) to the destination chain
+      3. Specify the **dest** parameter to indicate the destination chain for the message. The destination chain is the relay chain, so the value destination looks like this:
+            ```javascript
+            { "parents": 1, "interior": "Here" }
+            ```
+      4. Specify the **message** parameter to indicate the XCM version for the message. In this case, the value is `V2`
+      5. Click on **+ Add item** to construct the message to be executed
```

Comment:
The step `e` in the image is over **Submit Transaction** and not over **Add Item**